sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/multi/OfferDiscipline.java

```diff
@@ -0,0 +1,44 @@
+package com.mesosphere.sdk.scheduler.multi;
+
+import java.util.Collection;
+
+import com.mesosphere.sdk.scheduler.AbstractScheduler;
+import com.mesosphere.sdk.scheduler.SchedulerConfig;
+import com.mesosphere.sdk.scheduler.MesosEventClient.ClientStatusResponse;
+import com.mesosphere.sdk.storage.Persister;
+
+/**
+ * Interface used by {@link MultiServiceEventClient} to enforce the services which should or shouldn't be receiving
+ * offers.
+ */
+public interface OfferDiscipline {
+
+    /**
+     * Default implementation for creating an instance, based on the {@link SchedulerConfig}.
+     */
+    public static OfferDiscipline create(SchedulerConfig schedulerConfig, Persister persister) {
```

Comment:
I'd rather this logic not reside in the interface from which all disciplines will ultimately inherit.

It looks like we call this only in the Main.java for Hello World. Can you move it there?